src/components/PopularInstructors/PopularInstructors.tsx

```diff
@@ -0,0 +1,32 @@
+import { PopularInstructorsData } from "./PopularInstructorsData";
+import { PopularInstructorsCard } from "../PopularInstructorsCard/PopularInstructorsCard";
+import styles from "./PopularInstructors.module.scss";
+import Link from "next/link";
+
+export const PopularInstructors = () => {
+	return (
+		<section className={styles.popularInstructorsSection}>
+			<h2 className={styles.title}>Top Instructors of the Month</h2>
+			<div className={styles.instructorCards}>
+				{PopularInstructorsData.map((data) => {
+					return (
+						<PopularInstructorsCard
+							key={data.key}
```

Comment:
Cardlarda bir etkileşime sahip olmadığımız için data içerisinde key tanımlamak yerine direkt map'in index'i kullanılabilir